forge-gui/res/cardsfolder/upcoming/mind_spike.txt

```diff
@@ -0,0 +1,9 @@
+Name:Mind Spike
+ManaCost:B
+Types:Sorcery
+A:SP$ Reveal | ValidTgts$ Opponent | RevealAllValid$ Card.nonLand+nonCreature+TargetedPlayerCtrl | RememberRevealed$ True | SubAbility$ DBDiscard | StackDescription$ SpellDescription | SpellDescription$ Target opponent reveals each noncreature, nonland card in their hand. You choose a card revealed this way. That player discards that card. You lose 2 life. If they didn't reveal a card this way, you draw a card.
+SVar:DBDiscard:DB$ Discard | ValidTgts$ Opponent | Mode$ Defined | RevealNumber$ X | NumCards$ 1 | SubAbility$ DBLoseLife

```

Comment:
There is only one target above already

Reply:
You want `Defined$ Targeted` instead of `ValidTgts` in this line